src/main/java/io/vertx/ext/asyncsql/package-info.java

```diff
@@ -185,7 +185,9 @@
  *   "password" : <your-password>,
  *   "database" : <name-of-your-database>,
  *   "charset" : <name-of-the-character-set>,
- *   "queryTimeout" : <timeout-in-milliseconds>
+ *   "queryTimeout" : <timeout-in-milliseconds>,
+ *   "sslmode" : <"disable"|"prefer"|"require"|"verify-ca"|"verify-full">,
+ *   "sslrootcert" : <path to file with certificate>
```

Comment:
But I won't change "verify-ca"  and "verify-full" as it's direct values passed to underlying library.